### ISSUE TITLE
`pip wheel` unable to find LICENSE file

### ISSUE THREAD

**Issue by cooperlees** [ecederstrand/exchangelib#437]
I build my own wheels from sdists so that I can link with my own versions of C libraries etc and I had trouble building exchange lib:
```
running install_scripts
  error: [Errno 2] No such file or directory: 'LICENSE'
  Running setup.py bdist_wheel for exchangelib: finished with status 'error'
```
I'm going to try adding a MANIFEST.in to explicitly include the LICENSE file and see if that helps. PR coming.

**Comment by ecederstrand** [comment]
Fixed by #438

**Comment by cooperlees** [comment]
Thanks!
